### PR TITLE
Setting up a Wi-Fi extender using a raspberry pi

### LINKED ISSUE #3665: Using the Raspberry Pi as a WiFi extender
### Proposed title of article

Using the Raspberry Pi as a WiFi extender.

### Introduction paragraph (2-3 paragraphs):

The Raspberry Pi is essential as it has significant features. It can easily be turned into a WiFi Extender as it has an in built WiFi as well as Bluetooth. A Raspberry Pi WiFi Extender increases the range of the WiFi network while running various tasks either remotely or at the office. The WiFi Extender has some variations from any given access point as it obtains its network connection from an adapter. However, configuration for the range extenders is easy but there are certain disadvantages. They require separate network SSID that one has to log into reducing the speed for the WiFi as compared to accessing the network from the main router.

### Key takeaways: 
1. Connecting your devices to the Raspberry Pi.
2. Extending the range of the network.
3. Setting up a WiFi Extender.
 
### References: 
N/A

### PR DESCRIPTION
**Pull request by eunice-wanjiku**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes

This closes #3665

### CODE REVIEW

**Comment by louisefindlay23** [comment]
@eunice-wanjiku, please remove the backticks from This closes issue #3665 to properly link your topic suggestion to this PR.

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

File: setting-up-a-wifi-extender-using-a-raspberry-pi/index.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+1. To modify the hostapd configuration;
+
+```sudo nano /etc/hostapd/hostapd.conf```
+
+2. Here we dictate how we can communicate and interact with the WLAN device by changing the following lines:
+
+ ssid= “ ” //here you enter the name of the Wi-Fi 
+
+wpa_passphrase=” ” //here you enter the password of the Wi-Fi
```
Comment (collins-a) [inline setting-up-a-wifi-extender-using-a-raspberry-pi/index.md:97]: Consider adding an image for this step

**Review by collins-a — CHANGES_REQUESTED** [review]
@eunice-wanjiku good job with this tutorial on setting up an extender using Pi. A couple of comments to consider and we should be good to go.

**Comment by eunice-wanjiku** [comment]
@collins-a This is done

**Review by collins-a — APPROVED** [review]
@eunice-wanjiku thank you for making the required changes

**Comment by eunice-wanjiku** [comment]
@collins-a It's been 9 days. Kindly update what's happening with this PR?

**Pull request merged**